src/NServiceBus.Core/App_Packages/Particular.Licensing/FindActiveLicense/ActiveLicense.cs

```diff
@@ -20,15 +20,22 @@ public static ActiveLicenseFindResult Find(string applicationName, params Licens
             var licenseSourceResultToUse = LicenseSourceResult.DetermineBestLicenseSourceResult(results.ToArray());
             if (licenseSourceResultToUse != null)
             {
-                activeLicense.Report.Add($"Selected active license from {licenseSourceResultToUse.Location}");
+                var selectedLicenseReportItem = $"Selected active license from {licenseSourceResultToUse.Location}";
+                activeLicense.Report.Add(selectedLicenseReportItem);
```

Comment:
Yes, that would do that, but I wanted to introduce this change in a non-breaking way, because I still intend to push these changes upstream so they're in the licensing source package.